js_modules/dagster-ui/packages/ui-core/src/schedules/ScheduleCronTag.tsx

```diff
@@ -0,0 +1,32 @@
+import {Tag, Tooltip} from '@dagster-io/ui-components';
+import styled from 'styled-components';
+
+import {humanCronString} from './humanCronString';
+
+interface Props {
+  cronSchedule: string;
+  executionTimezone: string | null;
+}
+
+export const ScheduleCronTag = (props: Props) => {
```

Comment:
Maybe just CronTag? since Cron is inherently a schedule already? Just thinking we might want to render human readable cron tags outside of the context of schedules